modules/installing-ocp-agent-encrypt.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent-based-installer.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent-encrypt_{context}"]
+= Optional: Encrypting the disk
+
+Use this procedure to encrypt your disk or partition while installing {product-title} with the Agent-based Installer.
+
+If you do not want to encrypt your disk or partition, proceed to the next section.
+
+.Prerequisites
+
+* You have created and configured the `install-config.yaml` and `agent-config.yaml` files, unless you are using ZTP manifests.
+
+* You have placed the `openshift-install` binary in a directory that is on your PATH.
+
+.Procedure
+
+. Use the following command to generate ZTP cluster manifests:
++
+[source,terminal]
+----
+$ openshift-install agent create cluster-manifests --dir <installation_directory>
+----
++
+[IMPORTANT]
+====
+If you have created the `install-config.yaml` and `agent-config.yaml` files, those files will be deleted and replaced by the cluster manifests generated through this command.
+
+Any configurations made to the `install-config.yaml` and `agent-config.yaml` files are imported to the ZTP cluster manifests created by running the `openshift-install agent create cluster-manifests` command.
+====
++
+[NOTE]
+====
+If you have already generated ZTP manifests, skip this step.
+====
+
+. Navigate to the `cluster-manifests` directory:
++
+[source,terminal]
+----
+$ cd <installation_directory>/cluster-manifests
+----
+
+. Add the following section to the `agent-cluster-install.yaml` file:
++
+[source,yaml]
+----
+diskEncryption:
+    enableOn: all <1>
+    mode: tpmv2 <2>
```

Comment:
Maybe the example mode here should be `tang`, otherwise it's inconsistent with the next line.